src/renderer/services/eth2/client/logger.ts

```diff
@@ -1,41 +1,123 @@
+
+                            return (
+                                `${info.timestamp} ${info.level.padStart(infoPad)}: ${info.message}`
+                            );
+                        }),
+                        format.colorize(),
+                    )
+                })
+            ],
+            exitOnError: false
+        });
+        //@ts-ignore
+        this._level = LogLevel[options.level];
+        this._silent = false;
     }
 
     public debug(message: string | object, context?: object): void {
-        electronLogger.debug(message, context);
+        this.createLogEntry(LogLevel.debug, message, context);
     }
 
-    public error(message: string | object, context?: object): void {
-        electronLogger.error(message, context);
+    public info(message: string | object, context?: object): void {
+        this.createLogEntry(LogLevel.info, message, context);
     }
 
     public important(message: string | object, context?: object): void {
-        electronLogger.warn(message, context);
+        this.createLogEntry(LogLevel.info, chalk.red(message as string), context);
     }
 
-    public info(message: string | object, context?: object): void {
-        electronLogger.info(message, context);
+    public error(message: string | object, context?: object): void {
+        this.createLogEntry(LogLevel.error, message, context);
     }
 
-    public silly(message: string | object, context?: object): void {
-        electronLogger.verbose(message, context);
+    public warn(message: string | object, context?: object): void {
+        this.createLogEntry(LogLevel.warn, message, context);
     }
 
     public verbose(message: string | object, context?: object): void {
-        electronLogger.verbose(message, context);
+        this.createLogEntry(LogLevel.verbose, message, context);
     }
 
-    public warn(message: string | object, context?: object): void {
-        electronLogger.warn(message, context);
+    public silly(message: string | object, context?: object): void {
+        this.createLogEntry(LogLevel.silly, message, context);
+    }
+
+    public set level(level: LogLevel) {
+        this.winston.level = LogLevel[level];
+        this._level = level;
+    }
+
+    public get level(): LogLevel {
+        return this._level;
+    }
+
+    public set silent(silent: boolean) {
+        this._silent = silent;
+    }
+
+    public get silent(): boolean {
+        return this._silent;
+    }
+
+    public child(options: ILoggerOptions): ApiLogger {
+        const logger = Object.create(ApiLogger.prototype);
+        const winston = this.winston.child({namespace: options.module});
+        return Object.assign(logger, {
+            winston,
+            _level: options.level,
+            _silent: this._silent
+        });
+    }
+
+    public get stream(): PassThrough {
+        return this._stream;
+    }
+
+    private createLogEntry(level: LogLevel, message: string | object, context: object = {}): void {
```

Comment:
I see you've copied this from lodestar, might wanna check new implementation which fixes log levels. I'+ll take a look in winston logger so we expose method to pass custom tranfports